build/patches/Enable-crash-reporting-ui.patch

```diff
@@ -192,21 +441,82 @@ diff --git a/components/minidump_uploader/android/java/src/org/chromium/componen
 -            } finally {
 -                connection.disconnect();
 -            }
+-        } catch (IOException | RuntimeException e) {
 +
-+            File outputDir = new File(PathUtils.getDownloadsDirectory());
-+            File destination = new File(outputDir, fileToUpload.getName() + ".zip");
-+
-+            InputStream minidumpInputStream = new FileInputStream(fileToUpload);
-+            OutputStream out = new BufferedOutputStream(
-+                    new FileOutputStream(destination));
-+            ZipOutputStream requestBodyStream =
-+                    new ZipOutputStream(out);
-+            requestBodyStream.putNextEntry(new ZipEntry(fileToUpload.getName()));
-+            streamCopy(minidumpInputStream, requestBodyStream);
-+            return Result.success(destination.getPath());
-         } catch (IOException | RuntimeException e) {
++            return Result.success(fileToUpload.getPath());
++        } catch (RuntimeException e) {
              return Result.failure(e.toString());
          }
+     }
+diff --git a/components/upload_list/text_log_upload_list.cc b/components/upload_list/text_log_upload_list.cc
+--- a/components/upload_list/text_log_upload_list.cc
++++ b/components/upload_list/text_log_upload_list.cc
+@@ -27,7 +27,7 @@ constexpr char kJsonLogKeyState[] = "state";
+ constexpr char kJsonLogKeySource[] = "source";
+ 
+ std::vector<std::string> SplitIntoLines(const std::string& file_contents) {
+-  return base::SplitString(file_contents, base::kWhitespaceASCII,
++  return base::SplitString(file_contents, "\n",
```

Comment:
How come this change?

Reply:
practically in the `uploads.log` file there are 3 elements divided by "," arranged on lines

`<uploadtime>,<local_id>,<capturetime>`

in local_id I put the path which have spaces ..

```
1620458520,/data/user/0/org.chromium.chrome.stable/cache/Crash Reports/73d35758-26ea-441f-9ab6-97ff7d3c8afa.forced30226.try0,97ff7d3c8afa
```

I could remove the path and not touch that, for the code currently the path is useless

congratulations, nice review

Reply:
Why don't we store the relative path there? The prefix `/data/user/0/org.chromium.chrome.stable/cache/Crash Reports` never changes; reason: I am concerned it might break somewhere else.

Reply:
> congratulations, nice review

You are welcome! Thanks. I do not think I am doing my best but trying.

Reply:
fixed in https://github.com/bromite/bromite/pull/1105/commits/2eacdb9cfab05dd5fcbee058ce545fdd71253fe8